deployment/docker/REQUIREMENTS.txt

```diff
@@ -1,5 +1,5 @@
 psycopg2-binary
-Django>=2.0.2
+Django==2.1.8
```

Comment:
why not using 2.2?

Reply:
I was trying to fix the Travis builds fail during during migration due to:
`AttributeError: /usr/lib/libgdal.so.1: undefined symbol: GDALGetMetadataDomainList`
https://docs.djangoproject.com/en/3.0/releases/2.2/
https://stackoverflow.com/questions/55877882/beanstalk-migration-failing-for-geodjango

That worked - but I pinned it to latest 2.2 now.